terraform/modules/benchmark/service/lotus_bench.rb

```diff
@@ -41,11 +41,7 @@ def clone_command
   end
 
   def checkout_command
-    if @chain == 'mainnet'
-      exec_command(%w[git checkout releases])
-    else
-      exec_command(%w[git checkout master])
-    end
+    exec_command(%w[git checkout release/v1.23.3])
```

Comment:
I think such thing would be nicer to have in terraform variables (with `v1.23.3` as default)

Reply:
Good idea, will move it out!

Reply:
Done.